### ISSUE TITLE
TrailingComma false positive in heredoc

### ISSUE THREAD

**Issue by jimmycuadra** [rubocop/rubocop#764]
Given the code:

``` ruby
route(/^auth\s+list/, :list, command: true, restrict_to: :admins, help: {
  "auth list [GROUP]" => <<-HELP.chomp
List authorization groups and the users in them. If GROUP is supplied, only lists that group.
HELP
})
```

Rubocop reports an offense:

```
C: TrailingComma: Avoid comma after the last item of a hash.
List authorization groups and the users in them. If GROUP is supplied, only lists that group.
                                                                     ^
```

The comma appears inside a heredoc and should not be treated as following the final key/value pair of the hash.

Rubocop version details:

```
$ rubocop -V
0.17.0 (using Parser 2.1.4, running on ruby 2.1.0 x86_64-darwin13.0)
```
